library/core/src/ptr/mod.rs

```diff
@@ -2290,3 +2295,20 @@ pub macro addr_of($place:expr) {
 pub macro addr_of_mut($place:expr) {
     &raw mut $place
 }
+
+#[cfg(kani)]
+#[unstable(feature="kani", issue="none")]
+mod verify {
+    use crate::fmt::Debug;
+    use super::*;
+    use crate::kani;
+
+    #[kani::proof_for_contract(read_volatile)]
+    pub fn check_read_u128() {
+        let val = kani::any::<u16>();
```

Comment:
I imagine this is just a place holder, so we don't want to explore other types.

Reply:
yes, this was just a proof of concept.